README.md

```diff
@@ -1,16 +1,49 @@
 # React Design Patterns
 
-This repository holds the source code for some [PX Blue's design patterns](https://pxblue.github.io/patterns), written in React. 
+This repository holds the source code examples for [PX Blue design patterns](https://pxblue.github.io/patterns), written in React. The code is organized to make it easy to copy and paste the examples into your own projects.
 
-> NOTE: once this goes public, we will be able to provide deeplinks into each pattern using CodeSandbox.
+## Current Patterns
+
+| Pattern                                                                        | Description                                                                                                                                                                                  | Interactive Demo                                                                                                                                                                 |
+| ------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Action List](https://pxblue.github.io/patterns/lists)                         | A list with actions available on individual list items                                                                                                                                       | [CodeSandbox](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/action-list/index.tsx&initialpath=action-list)                      |
```

Comment:
These links are still opening App.tsx for me. I think you need to update `?file=` to `?module=`, e.g.:
`?module=%2Fsrc%2Fpages%2Faction-list.tsx`